### PR TITLE
New CTDMO-S cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

**Review by sauuyer — APPROVED** [review]
checked against .cal files - all look good

File: calibration/CTDMOS/CGINS-CTDMOS-13436__20240702.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+37-13436,CC_ptca0,5.218506e+005,
+37-13436,CC_ptca1,4.764960e+000,
+37-13436,CC_ptca2,-3.477787e-001,
+37-13436,CC_ptcb0,2.474150e+001,
+37-13436,CC_ptcb1,1.000000e-004,
+37-13436,CC_ptcb2,0.000000e+000,
+37-13436,CC_ptempa0,-8.352358e+001,
+37-13436,CC_ptempa1,4.732327e-002,
+37-13436,CC_ptempa2,-2.163752e-007,
+37-13436,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13436__20240702.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13562__20240702.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+37-13562,CC_ptca0,5.241525e+005,
+37-13562,CC_ptca1,-4.465616e+000,
+37-13562,CC_ptca2,-5.263758e-002,
+37-13562,CC_ptcb0,2.518925e+001,
+37-13562,CC_ptcb1,-3.500000e-004,
+37-13562,CC_ptcb2,0.000000e+000,
+37-13562,CC_ptempa0,-8.332999e+001,
+37-13562,CC_ptempa1,4.687448e-002,
+37-13562,CC_ptempa2,-2.402975e-007,
+37-13562,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13562__20240702.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13576__20240726.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-13576,CC_ptca0,5.221711e+005,
+37-13576,CC_ptca1,-3.855054e+000,
+37-13576,CC_ptca2,-8.228991e-002,
+37-13576,CC_ptcb0,2.506237e+001,
+37-13576,CC_ptcb1,-1.525000e-003,
+37-13576,CC_ptcb2,0.000000e+000,
+37-13576,CC_ptempa0,-7.670372e+001,
+37-13576,CC_ptempa1,5.031888e-002,
+37-13576,CC_ptempa2,-6.462962e-007,
+37-13576,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13576__20240726.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13587__20240531.csv [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+37-13587,CC_ptca0,5.248174e+005,
+37-13587,CC_ptca1,5.145868e+000,
+37-13587,CC_ptca2,-2.357581e-002,
+37-13587,CC_ptcb0,2.543863e+001,
+37-13587,CC_ptcb1,-7.500000e-005,
+37-13587,CC_ptcb2,0.000000e+000,
+37-13587,CC_ptempa0,-8.605245e+001,
+37-13587,CC_ptempa1,4.604725e-002,
+37-13587,CC_ptempa2,-1.409392e-007,
+37-13587,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13587__20240531.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13602__20240531.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+37-13602,CC_ptca0,5.247005e+005,
+37-13602,CC_ptca1,8.915399e+000,
+37-13602,CC_ptca2,-9.663688e-002,
+37-13602,CC_ptcb0,2.501362e+001,
+37-13602,CC_ptcb1,1.246883e-004,
+37-13602,CC_ptcb2,0.000000e+000,
+37-13602,CC_ptempa0,-8.332130e+001,
+37-13602,CC_ptempa1,4.769674e-002,
+37-13602,CC_ptempa2,-3.418346e-007,
+37-13602,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13602__20240531.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13615__20240703.csv [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+37-13615,CC_ptca0,5.242540e+005,
+37-13615,CC_ptca1,1.971330e+000,
+37-13615,CC_ptca2,-1.195466e-001,
+37-13615,CC_ptcb0,2.506664e+001,
+37-13615,CC_ptcb1,9.273183e-004,
+37-13615,CC_ptcb2,0.000000e+000,
+37-13615,CC_ptempa0,-7.902935e+001,
+37-13615,CC_ptempa1,4.861654e-002,
+37-13615,CC_ptempa2,-3.817817e-007,
+37-13615,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13615__20240703.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13618__20240606.csv [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+37-13618,CC_ptca0,5.247267e+005,
+37-13618,CC_ptca1,2.935519e+000,
+37-13618,CC_ptca2,-5.512541e-002,
+37-13618,CC_ptcb0,2.513038e+001,
+37-13618,CC_ptcb1,-3.250000e-004,
+37-13618,CC_ptcb2,0.000000e+000,
+37-13618,CC_ptempa0,-8.453998e+001,
+37-13618,CC_ptempa1,4.681853e-002,
+37-13618,CC_ptempa2,-1.928292e-007,
+37-13618,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13618__20240606.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13619__20240702.csv [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+37-13619,CC_ptca0,5.249471e+005,
+37-13619,CC_ptca1,8.394614e-001,
+37-13619,CC_ptca2,-8.439526e-002,
+37-13619,CC_ptcb0,2.513538e+001,
+37-13619,CC_ptcb1,-1.325000e-003,
+37-13619,CC_ptcb2,0.000000e+000,
+37-13619,CC_ptempa0,-7.735017e+001,
+37-13619,CC_ptempa1,4.885861e-002,
+37-13619,CC_ptempa2,-3.627457e-007,
+37-13619,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13619__20240702.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13625__20240602.csv [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+37-13625,CC_ptca0,5.241713e+005,
+37-13625,CC_ptca1,6.677514e+000,
+37-13625,CC_ptca2,-1.380497e-001,
+37-13625,CC_ptcb0,2.498900e+001,
+37-13625,CC_ptcb1,-2.000000e-004,
+37-13625,CC_ptcb2,0.000000e+000,
+37-13625,CC_ptempa0,-8.262130e+001,
+37-13625,CC_ptempa1,4.656471e-002,
+37-13625,CC_ptempa2,-2.132508e-007,
+37-13625,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13625__20240602.csv:25]: looks good

File: calibration/CTDMOS/CGINS-CTDMOS-13635__20240710.csv [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+37-13635,CC_ptca0,5.243114e+005,
+37-13635,CC_ptca1,-6.127623e+000,
+37-13635,CC_ptca2,-8.609187e-003,
+37-13635,CC_ptcb0,2.516800e+001,
+37-13635,CC_ptcb1,-4.000000e-004,
+37-13635,CC_ptcb2,0.000000e+000,
+37-13635,CC_ptempa0,-7.570829e+001,
+37-13635,CC_ptempa1,4.839003e-002,
+37-13635,CC_ptempa2,-2.998895e-007,
+37-13635,CC_p_range,160,
```
Comment (cdobs) [inline calibration/CTDMOS/CGINS-CTDMOS-13635__20240710.csv:25]: looks good

**Review by cdobs — APPROVED** [review]

**Pull request merged**